### PR TITLE
New react goodness!

### LINKED ISSUE #20: Make photo cards more like 'Pintrest'

### LINKED ISSUE #21: Cats can be filtered by tag

### LINKED ISSUE #28: New icons to be incorporated into V2
Used to update favicon and possibly used in app

This cat is white-fill :point_down: 
cat-fill-white
:point_up: 
cat-outline
cat-fill

### PR DESCRIPTION
**Pull request by ghost**
- [x] New Favicons/Icons
- [x] Iterate through JSON
- [x] Fix like button
- [x] Add local storage
- [x] New likes saved in local storage
- [x] Footer
- [x] Save uploaded cats to local storage
- [x] Filter by tag
- [x] Filter by input
- [x] New UI changes and features
- [x] Remove white space surrounding tags
- [x] Format the search terms to lower or upper

closes #28
closes #21
closes #20

### CODE REVIEW

File: app/components/CardAction.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {render} from 'react-dom';
+
+class CardContent extends React.Component {
```
Comment (Bouncey) [inline app/components/CardAction.js]: Should this not be CardAction?

File: app/components/CardAction.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      <div className="card-action">
+        <button className="waves-effect waves-light waves-blue blue darken-3 btn like">
+          <i className="fa fa-thumbs-up"></i>
+          <span className="likes_number"> {this.props.likes}</span>
+        </button>
+      </div>
+    );
+  }
+}
+export default CardContent;
```
Comment (Bouncey) [inline app/components/CardAction.js]: And CardAction here?

**Comment by Bouncey** [comment]
Need to remove white space surrounding the tags and toLowercase the search terms

**Comment by Bouncey** [comment]
Pull it down and take a look! :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat: :smiley_cat:

**Comment by Bouncey** [comment]
Need some more validation for title and url

**Comment by atjonathan** [comment]
### **LGTM :+1:**
Thanks again @Bouncey

**Pull request merged**